Asset/src/V1/Gapic/AssetServiceGapicClient.php

```diff
@@ -761,18 +761,24 @@ public function analyzeMove(
      *
      * @throws ApiException if the remote call fails
      */
-    public function batchGetAssetsHistory($parent, $contentType, $readTimeWindow, array $optionalArgs = [])
+    public function batchGetAssetsHistory($parent, array $optionalArgs = [])
```

Comment:
This is a BC-Breaking change, and `google/cloud-asset` is a GA library, so we'll need to figure out how to handle this before merging